packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -368,6 +369,45 @@ const TOO_MANY_SNAPSHOT_COLUMNS = [
   },
 ]
 
+const GUEST_TOOLS_COLUMNS = [
+  {
+    name: _('vmNameLabel'),
+    itemRenderer: vm => <Link to={`vms/${vm.id}`}>{vm.name_label}</Link>,
+    sortCriteria: vm => vm.name_label,
+  },
+  {
+    name: _('vmNameDescription'),
+    itemRenderer: vm => vm.name_description,
+    sortCriteria: vm => vm.name_description,
+  },
+  {
+    name: _('vmContainer'),
+    itemRenderer: vm => <VmColContainer id={vm.$container} />,
+  },
+  {
+    default: true,
+    name: _('guestToolStatusColumn'),
+    itemRenderer: vm => {
+      if (!vm.pvDriversDetected) {
+        return _('noToolsDetected')
+      }
+      if (!vm.managementAgentDetected) {
+        return _('managementAgentNotDetected')
+      }
+
+      const version =
+        getDefined(() => vm.pvDriversVersion.split('.')[0]) > 0
+          ? vm.pvDriversVersion
+          : ''
+
+      return _('managementAgentOutOfDate', {
+        version,
+      })
+    },
+    sortOrder: 'desc',
```

Comment:
If you want this column to be sortable, you need to declare a `sortCriteria`. It could be as simple as:
```suggestion
    sortCriteria: vm =>
      !vm.pvDriversDetected ? 0 : !vm.managementAgentDetected ? 1 : 2,
```